src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/ubuntu14.04_debug_prtest/3181/consoleFull#1170600644a82fefab-f698-416f-8fca-58544c94cd4e

`/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug_prtest/src/Common/src/Interop/Unix/System.Native/Interop.Socket.cs(16,81): CS0104: 'SocketType' is an ambiguous reference between 'System.Net.Internals.SocketType' and 'System.Net.Sockets.SocketType' `